Custom jinja Undefined class for handling nested undefined attributes (#51768)
ansible/ansible: docs/docsite/rst/porting_guides/porting_guide_2.8.rst

To achieve the results of notifying a single name but running mulitple handlers, utilize ``include_tasks``, or ``listen`` :ref:`handlers`.

Jinja Undefined values
----------------------

> Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an exception. This means that you no longer need to jump

Reviewer: In the current behavior, is there an error message when Ansible throws the exception? If so, it would be great to include that here so future users searching for that error will find this porting guide entry (and maybe be encouraged to upgrade).

Author: I changed the word `exception` to `error`, since the exception is an internal implementation detail. The actual error you get is `The task includes an option with an undefined variable. The error was: 'foo' is undefined`, but I'm not sure how to fit that in there.

Author: I further changed it to "rather than throwing an error immediately".

> Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an exception. This means that you no longer need to jump

Reviewer: ```suggestion
Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an exception. This means you can
```

> through as many hoops when accessing a value in a nested data structure when you don't know if the intermediate values are defined.

Reviewer: ```suggestion
access a value in a nested data structure even when you don't know if the intermediate values are defined.
```
Does this convey the same meaning? If so, I'd prefer this phrasing, which focuses on the positive.

Author: I changed it to use this wording, with a minor change.
